app/views/settings/preferences/other/show.html.haml

```diff
@@ -10,13 +10,19 @@
   .fields-group
     = f.input :setting_noindex, as: :boolean, wrapper: :with_label
 
+  .fields-group
+    = f.input :setting_norss, as: :boolean, wrapper: :with_label
+
   .fields-group
     = f.input :setting_aggregate_reblogs, as: :boolean, wrapper: :with_label, recommended: true
 
   - unless Setting.hide_followers_count
     .fields-group
       = f.input :setting_hide_followers_count, as: :boolean, wrapper: :with_label, glitch_only: true
 
+  .fields-group
+    = f.input :setting_norss, as: :boolean, wrapper: :with_label, label: t('admin.settings.default_norss.title'), hint: t('admin.settings.default_norss.desc_html')
+
```

Comment:
```suggestion
```

This setting is exposed to every user.
It belong in `app/views/admin/settings/edit.html.haml`

Reply:
What the issue? It's a per-user option. Checking the box would not disable rss feeds for all users on a server. 

Furthermore, the file you mentioned doesn't exist.

Reply:
Ah, I just compared it to the hometown commit.
hometown still uses 3.5.x and a lot changed in 4.0.x.
It should probably be in `app/views/admin/settings/other/show.html.haml`  or `app/views/admin/settings/discovery/show.html.haml` even though both don't really fit.

The issue is, that that part isn't a user option, but an admin setting to set a default for \*every\* user.
Or at least that's what it's supposed to do.

Reply:
I see that's what the setting description did - that should change. This PR seems to only bring the per-user choice over, I missed the admin override.

Reply:
I think letting admins change the default without having to manually edit `settings.yml` is a good thing.
Sadly upstream also removed that option for noindex.

Reply:
[10372c8](https://github.com/glitch-soc/mastodon/pull/2002/commits/10372c8d331ca5adf08a9b72790b684d5e19f613) resolves this.